ocp_resources/metallb.py

```diff
@@ -0,0 +1,54 @@
+# API reference: https://metallb.universe.tf/apis/#metallb.io/v1beta1
+
+from ocp_resources.resource import NamespacedResource
+
+
+class MetalLB(NamespacedResource):
```

Comment:
`MetalLB` I can't find any resource with such kind in MetalLB docs.

Reply:
I did asked people who contribute to the MetalLB, there is no specific document for `kind: MetalLB`.

Reply:
wdyt, can we link to resource definition until docs are updated?
https://github.com/metallb/metallb-operator/blob/main/api/v1beta1/metallb_types.go

Reply:
Thanks Sarah, I was finding the similar definition. @myakove is it okay to add above link ?

Reply:
I guess this is our only option for now

Reply:
Okay I will update PR.

Reply:
Upated.